packages/@angular/cli/blueprints/class/index.ts

```diff
@@ -30,7 +30,7 @@ export default Blueprint.extend({
 
     options.spec = options.spec !== undefined ?
       options.spec :
-      this.project.ngConfigObj.get('defaults.spec.class');
+      this.project.ngConfigObj.get('defaults.class.spec');
```

Comment:
Please use an alias for these, and a warning if the alias is used.

See [here](https://github.com/angular/angular-cli/blob/0dc2200004a5621a1576624b527e529a1bb1ecea/packages/%40angular/cli/models/config.ts#L50) for an example.